docs/faq/repositories/how-do-i-reanalyze-my-repository.md

```diff
@@ -1,5 +1,5 @@
 # How do I reanalyze my repository?
 
-Click on the latest commit from your list in the **Commit** view and click to reanalyze as the latest commit includes all the code of all the PRs.
+Open the latest commit on the [Commits page](../../repositories/commits.md) and click the button next to **Analyzed**, as the latest commit includes all the code from all the pull requests.
```

Comment:
a comment not related to this change, but I don't think this sentence is accurate:

> the latest commit includes all the code from all the pull requests

it depends on the target branch of the pull request, and reanalyzing the latest commit does not automatically reanalyze the pull requests results.

Reply:
I'm going to open a separate pull request to fix this. :+1: